frontend/src/components/svgs/index.js

```diff
@@ -130,10 +130,9 @@ const LaptopIcon = () => (
   </svg>
 );
 
-const MailIcon = () => (
+const MailIcon = ({ className = "scale-50" }) => (
```

Comment:
```suggestion
const MailIcon = ({ className = "scale-100" }) => (
```
you resize the original use of it.  could you change it to this?